cou/logging.py

```diff
@@ -76,7 +76,8 @@ def setup_logging(log_level: str = "INFO") -> None:
 
     root_logger.addHandler(log_file_handler)
     root_logger.addHandler(console_handler)
-    logger.info("Logs of this execution can be found at %s", file_name)
+    progress_indicator.start("Configuring logging...")
```

Comment:
This should be put at the beginning of the function. Otherwise we would not see the indication if something goes wrong during logger configuration.